### ISSUE TITLE
sys.getcheckinterval is deprecated

### ISSUE THREAD

**Issue by dojeda** [tqdm/tqdm#319]
Just a heads up: In Python 3.5, tqdm is currently generating this warning:

```
lib/python3.5/site-packages/tqdm/_tqdm.py:65: DeprecationWarning: sys.getcheckinterval() and sys.setcheckinterval() are deprecated.  Use sys.setswitchinterval() instead.
	  sys.setcheckinterval(100)
```

Which I guess can be easily solved by using `sys.setswitchinterval(100)`. But I'll let you guys decide that, since you know exactly what's going on your internal threads.

**Comment by lrq3000** [comment]
Thanks for the heads up :) No pb about this command, we can even remove it
as it is setting thn same value as Python's default anyway, i just added it
to make sure the value doesn't change in future python releases (as this
would break our unit tests since this is essentially the thread's refresh
rate). We will add a compatibility layer :)

Le 6 Déc. 2016 17:47, "Casper da Costa-Luis" <notifications@github.com> a
écrit :

**Comment by casperdcl** [comment]
closed by #326
